pkgs/tools/security/ssh-mitm/default.nix

```diff
@@ -3,36 +3,22 @@
 , python3
 }:
 
-let
-  py = python3.override {
-    packageOverrides = self: super: {
-      paramiko = super.paramiko.overridePythonAttrs (oldAttrs: rec {
-        version = "3.1.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          hash = "sha256-aVD6ymgZrNMhnUrmlKI8eofuONCE9wwXJLDA27i3V2k=";
-        };
-        patches = [ ];
-        propagatedBuildInputs = oldAttrs.propagatedBuildInputs ++ [ python3.pkgs.icecream ];
-      });
-    };
-  };
-in
-with py.pkgs;
 
-buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "ssh-mitm";
-  version = "3.0.2";
+  version = "4.1.1";
   format = "setuptools";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "refs/tags/${version}";
     hash = "sha256-koV7g6ZmrrXk60rrDP8BwrDZk3shiyJigQgNcb4BASE=";
+    # Correct hash does not seems to build ??
+    #hash = "sha256-Uf1B7oEZyNWj4TjrLvEfFdxsvsGeMLXFsSdxGLUV4ZU=";
```

Comment:
This is the proper hash of the package but I'm running into a failure : 
```
❯ nix-build -A ssh-mitm
this derivation will be built:
  /nix/store/2yl03yllx3z5w9bd8bfrp6q0wq9mrfwa-ssh-mitm-4.1.1.drv
building '/nix/store/2yl03yllx3z5w9bd8bfrp6q0wq9mrfwa-ssh-mitm-4.1.1.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Running phase: unpackPhase
unpacking source archive /nix/store/rksl0phb64vhv1l4yq0iw2shb8vwgpl1-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/sshmitm/workarounds/transport.py
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing setuptoolsBuildPhase
Finished executing setuptoolsBuildPhase
Running phase: installPhase
Executing pypaInstallPhase
/nix/store/dd3gsp3cvl6l641h4vcs86l9frznxsjx-pypa-install-hook/nix-support/setup-hook: line 8: pushd: dist: No such file or directory
error: builder for '/nix/store/2yl03yllx3z5w9bd8bfrp6q0wq9mrfwa-ssh-mitm-4.1.1.drv' failed with exit code 1;
       last 10 log lines:
       > Running phase: patchPhase
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > Executing setuptoolsBuildPhase
       > Finished executing setuptoolsBuildPhase
       > Running phase: installPhase
       > Executing pypaInstallPhase
       > /nix/store/dd3gsp3cvl6l641h4vcs86l9frznxsjx-pypa-install-hook/nix-support/setup-hook: line 8: pushd: dist: No such file or directory
       For full logs, run 'nix log /nix/store/2yl03yllx3z5w9bd8bfrp6q0wq9mrfwa-ssh-mitm-4.1.1.drv'.
```

When using the original hash, the build is a success.

Reply:
The build system is now `hatchling` and no `setuptools`.

```
hash = "sha256-Uf1B7oEZyNWj4TjrLvEfFdxsvsGeMLXFsSdxGLUV4ZU=";
```

should work then.